test/kwc-building-listing_test.html

```diff
@@ -25,6 +72,56 @@
           assert.equal(element.is, 'kwc-building-listing');
         });
       });
+
+      suite('ready', function () {
+            var element;
+            setup(function () {
+                element = fixture('basic');
+                element.challenges = challenges;
+            });
+            test('instantiating the element works', function () {
+                assert.equal(element.is, 'kwc-building-listing');
+            });
```

Comment:
This duplicates the one above doesn't it?

Reply:
Well, as each suite sets up the component independently and maybe in a different manner I though it might be worthwhile to check is still initiates correctly in each teat suite. Maybe this is not necessary or maybe not for this simple example, but it could be good practice in general. 
Open to other opinions though, of course.

Reply:
I wonder if we can write a suite that will be executed before all the specific tests. So we can always run the "basic" tests before running the specific ones. I do agree it's important to test if the very basic still works.